index.ts

```diff
@@ -186,6 +189,19 @@ export function createProxyCache<
     },
   };
 
+  const fetchGuild = async (id: bigint) => {
+    // Return nothing if we don't want fetching
+    if (!options.fetchGuildIfMissing) return null;
```

Comment:
just because of consistency
```suggestion
    if (!options.fetchGuildIfMissing) return undefined;
```